### PR TITLE
feat(http): return headers on response and fix bug

### PR DESCRIPTION
**Pull request by henri-hulski**

### CODE REVIEW

File: packages/cerebral-provider-http/src/utils.js [hunk 1]
```diff
@@ -64,3 +64,20 @@ export function convertObjectWithTemplates (obj, context) {
     return convertedObject
   }, {})
 }
+
+export function parseHeaders (rawHeaders) {
```
Comment (christianalfoni) [inline packages/cerebral-provider-http/src/utils.js]: Is it necessary to parse the headers? Can you not just grab them? Not quite sure if HTTP should be responsible for it :)

Comment (henri-hulski) [inline packages/cerebral-provider-http/src/utils.js]: > The getAllResponseHeaders( ) method returns all HTTP headers from the response of the XMLHttprequest. 
> HTTP headers are returned as a single string with each header line separated by a CR/LF pair and with each header name and header value separated by a ": " pair.

So I don't know if this is a bit useful.
Axios e.g. also parses the headers.

Comment (christianalfoni) [inline packages/cerebral-provider-http/src/utils.js]: Ah, they do? Sorry, I thought there was a native method for grabbing them. Okay, but looks good :)

Comment (henri-hulski) [inline packages/cerebral-provider-http/src/utils.js]: For axios see https://github.com/mzabriskie/axios/blob/master/lib/helpers/parseHeaders.js.

**Comment by christianalfoni** [comment]
Oh... sorry, I thought I was merging something else, oh well... it is approved :) Sorry @Guria

**Pull request merged**